externs/es6.js

```diff
@@ -394,6 +394,23 @@ function Int8Array(length, opt_byteOffset, opt_length) {}
 /** @type {number} */
 Int8Array.BYTES_PER_ELEMENT;
 
+/** 
+* @param {Array<number>} source
```

Comment:
I admit I haven't looked at this part of the spec but it sounds like this should really be Iterable<number> not Array<number>? In any case I'm fine with submitting this as Array for now and then we can change it to Iterable (or ArrayLike, or whatever it's supposed to really be) later. But, let's make it non-nullable (add the `!`)